java-bootcamp-bahia-t1/src/MailListSubject.java

```diff
@@ -0,0 +1,9 @@
+public interface MailListSubject {

```

Comment:
I used the observer pattern as needed represent a market manager (Observer) I received information from a MailList (Subject) according to different aspects that change.